### ISSUE TITLE
Verbose mode breaks some commands

### ISSUE THREAD

**Issue by DavidKelly999** [globus/globus-automate-client#111]
Using the -v flag breaks some commands (tested with 0.13.0b1 installed via pip)

For example, the commands below result in a traceback:
```
$ globus-automate flow get 23392c51-17b6-4558-bb56-dc915e1faf39 -v
$ globus-automate flow list -v
```

```
Traceback (most recent call last):
  File "/home/david/venvs/cfde/bin/globus-automate", line 8, in <module>
    sys.exit(app())
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/typer/main.py", line 497, in wrapper
    return callback(**use_params)  # type: ignore
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/flows.py", line 459, in flow_list
    RequestRunner(
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/rich_helpers.py", line 441, in run_and_render
    self.render(result)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/rich_helpers.py", line 421, in render
    Renderer(
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/rich_helpers.py", line 370, in render
    d = self.details_as_text
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/rich_helpers.py", line 309, in details_as_text
    return Text(self.result.details, style=self.detail_style)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/rich_helpers.py", line 247, in details
    return get_http_details(self.result)
  File "/home/david/venvs/cfde/lib/python3.8/site-packages/globus_automate_client/cli/helpers.py", line 40, in get_http_details
    base_request = result.data["request"]
KeyError: 'request'
```

**Comment by rudyardrichter** [comment]
Thanks for the report @DavidKelly999! It looks like the fix from #108 should resolve this once it's released.

**Comment by kurtmckee** [comment]
David, we've released 0.13.0b2, which should fix this issue. Please let us know if you encounter additional issues, and thank you again for testing this!
